packages/docs/docs/tailwind/enable-tailwind.md

```diff
@@ -0,0 +1,42 @@
+---
+image: /generated/articles-docs-tailwind-enable-tailwind.png
+title: enableTailwind()
+crumb: "@remotion/tailwind"
+---
+
+A function that modifies the default Webpack configuration to make the necessary changes to support TailwindCSS.
+
+```ts twoslash title="remotion.config.ts"
+import { Config } from "remotion";
+import { enableTailwind } from "@remotion/tailwind";
+
+Config.overrideWebpackConfig((currentConfiguration) => {
+  return enableTailwind(currentConfiguration);
+});
+```
+
+:::note
+Prior to `v3.3.39`, the option was called `Config.Bundling.overrideWebpackConfig()`.
+:::
+
+If you want to make other configuration changes, you can do so by doing them reducer-style:
+
+```ts twoslash title="remotion.config.ts"
+import { Config } from "remotion";
+import { enableTailwind } from "@remotion/tailwind";
+
+Config.overrideWebpackConfig((currentConfiguration) => {
+  const newConfig = enableTailwind(currentConfiguration);
+
+  return {
+    ...newConfig,
+    // Make other changes
+  };
```

Comment:
```suggestion
  const newConfig = {
  	// Make other change here
  	...currentConfiguration
  };

  return enableTailwind(currentConfiguration);
```

I believe we should ask users to make their changes before applying these override functions (just like middlewares in `next.config.mjs` as instructed by their docs) because in the `enableTailwind` function, we spread the `currentConfiguration` passed as the first params first, and then add the overrides for tailwind!

So, for example, (going with the current version), a user overrides the `module` key in the newConfig, then the tailwind middleware won't work, and they'd need to manually spread the module value from `newConfig`.

Reply:
Happy to take guidance on this, I don't feel like I've got firm opinions either way.

This example is near enough a copy from the `@remotion/skia` docs, so if we're happy updating it shall I do so in both places?